components/data-oriented-programming/code/java-bbe/flexible-typing.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: 'Be conservative in what you send, be liberal in what you accept'
-description: 'Ballerina employs "be conservative in what you send, be liberal in what you accept" by using structural types that support openness. <br><br>These types serve a dual purpose: enhancing static typing within programs and describing service interfaces accurately. While outgoing messages are tightly controlled to ensure protocol adherence, incoming data is handled with a degree of flexibility. The result is a balance of strictness and tolerance that enhances interoperability and resilience. This makes Ballerina a robust and adaptable choice for constructing cloud-native applications'
-image: 
+description: Ballerina employs be conservative in what you send, be liberal in what you accept by using structural types that support openness. <br><br>These types serve a dual purpose, enhancing static typing within programs and describing service interfaces accurately. While outgoing messages are tightly controlled to ensure protocol adherence, incoming data is handled with a degree of flexibility. The result is a balance of strictness and tolerance that enhances interoperability and resilience. This makes Ballerina a robust and adaptable choice for constructing cloud-native applications.
```

Comment:
I dont agree on removing the double quotes around `be conservative in what you send, be liberal in what you accept ` as it is the concept we need to highlight here and it is widely used phrase everywhere, so we are quoting it, not specific to this paragraph.

Reply:
Added the quotes back.